eclair-core/src/test/scala/fr/acinq/eclair/TestConstants.scala

```diff
@@ -56,9 +59,59 @@ object TestConstants {
     }
   }
 
-  def sqliteInMemory() = DriverManager.getConnection("jdbc:sqlite::memory:")
+  sealed trait TestDatabases {
+    val connection: Connection
+    def network(): NetworkDb
+    def audit(): AuditDb
+    def channels(): ChannelsDb
+    def peers(): PeersDb
+    def payments(): PaymentsDb
+    def pendingRelay(): PendingRelayDb
+    def getVersion(statement: Statement, db_name: String, currentVersion: Int): Int
+    def obtainLock(): Unit
+  }
+
+  case class TestSqliteDatabases(connection: Connection = sqliteInMemory()) extends TestDatabases {
+    override def network(): NetworkDb = new SqliteNetworkDb(connection)
+    override def audit(): AuditDb = new SqliteAuditDb(connection)
+    override def channels(): ChannelsDb = new SqliteChannelsDb(connection)
+    override def peers(): PeersDb = new SqlitePeersDb(connection)
+    override def payments(): PaymentsDb = new SqlitePaymentsDb(connection)
+    override def pendingRelay(): PendingRelayDb = new SqlitePendingRelayDb(connection)
+    override def getVersion(statement: Statement, db_name: String, currentVersion: Int): Int = SqliteUtils.getVersion(statement, db_name, currentVersion)
+    override def obtainLock(): Unit = ()
+  }
+
+  case object TestPsqlDatabases extends TestDatabases {
+    override val connection: Connection = psql()
+
+    import com.zaxxer.hikari.{HikariConfig, HikariDataSource}
+
+    val config = new HikariConfig
+    config.setJdbcUrl("jdbc:postgresql://localhost:5432/eclair")
```

Comment:
This another option?
https://dzone.com/articles/using-embedded-postgresql-databases-for-unit-testi

Reply:
Cool! I used `EmbeddedPG` for DB tests, so now there's no need to install actual Postgres to run tests.